Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:14.04
```

Comment:
You should be able to use one of the [Python base images](https://registry.hub.docker.com/_/python/). This would simplify the Dockerfile to:

```dockerfile
FROM python:3-onbuild

EXPOSE 5000

CMD ["python", "./rs.py"]
```